components/MaintainerCard/MaintainerCard.tsx

```diff
@@ -6,12 +6,8 @@ export interface IMaintainerCardProps {
 	maintainer: IMaintainer
 }
 
-export const MaintainerCard = ({
-	image,
-	fullName,
-	description,
-	githubLink,
-}: IMaintainer) => {
```

Comment:
oh, i think it was my PR 😬,
got a little confused here

Reply:
No, that's my fault. I overlooked your code.
Otherwise I'd have asked you to change it into
```
export const MaintainerCard = ({ maintainer }: IMaintainerCardProps) => {
	const { image, fullName, description, githubLink } = maintainer
```

Reply:
No worries, it’s easily done. I make mistakes all the time too. But it’s just as easily fixed 😀